src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataReorderSection.tsx

```diff
@@ -33,13 +34,27 @@ const ReleaseDataReorderSection = ({ releaseId, canUpdateRelease }: Props) => {
   const [reorderingFilters, setReorderingFilters] = useState<Subject>();
   const [reorderingIndicators, setReorderingIndicators] = useState<Subject>();
 
-  const handleSaveFilters = (updatedFilters: UpdateFiltersRequest) => {
+  const handleSaveFilters = (
+    subjectId: string,
+    updatedFilters: UpdateFiltersRequest,
+  ) => {
     setReorderingFilters(undefined);
-    // TO DO: send the update request.
+    adminTableBuilderService.updateFilters(
```

Comment:
Wouldn't it make more sense to toggle `setReorderingFilters` _after_ the service request has completed?

```ts
await adminTableBuilderService.updateFilters(...);
setReorderingFilters(undefined);
```

Similar applies for `handleSaveIndicators` below as well